metadata.go

```diff
@@ -0,0 +1,60 @@
+package main
+
+import (
+	"encoding/json"
+	"io/ioutil"
+	"net/http"
+	"strings"
+
+	"github.com/sirupsen/logrus"
+)
+
+func GetAllMetricMetadata(promMetaDataEndPoint string, metricsList map[string]MetricMetadata) {
```

Comment:
This feels pretty hacky. Is there anyway we could instead interface with the metrics object in memory instead of making an internal call?